src/hooks/useServices.js

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+
+import { GetServices } from "../services/ApiService";
+
+const useServices = () => {
+  const [serviceItems, setServiceItems] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    GetServices().then(response => {
+      setServiceItems(response);
+      setIsLoading(false);
+    });
+  }, []);
+
+  return [serviceItems, isLoading];
```

Comment:
this can use the same pattern as `useHasNetworkError`, where the object is returned instead of an array

Reply:
done